recipes/farmhash/all/conanfile.py

```diff
@@ -12,31 +11,26 @@ class farmhashConan(ConanFile):
     homepage = "https://github.com/google/farmhash"
     url = "https://github.com/conan-io/conan-center-index"
     settings = "os", "arch", "compiler", "build_type"
-
+    exports_sources = ["CMakeLists.txt"]
     options = {
         "shared": [True, False],
-        "fPIC": [True, False],
-        "no_builtin_expect": [True, False]
```

Comment:
Could you consider to preserve option "no_builtin_expect"?
Deleting options immediately seems to be not preferable in conan-center-index.

Please refer https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#can-i-remove-an-option-from-recipe.